data/nosql/mongo-init.js

```diff
@@ -16,33 +16,51 @@ db.customer_details.insert([
         "customer_id": 1,
```

Comment:
Not for part of this PR, but for this to match we'll need to convert these to `"1"`, since that's the type in `db.customer.inset(...)`

Reply:
It might actually be good to get the mismatched types to work (using integer to find a string, although I think that statement inserts it as float, maybe we insert it as an integer instead).  We in theory support some limited type coercion to use data of one type to find data of another type, but don't have a lot of examples of it. 

This is somewhat related to my type coercion commit here https://github.com/ethyca/fidesops/pull/194/commits/741c9dd34a2e5ed43e2333eb5ae3fb507f58dc57

Reply:
I've made them the same types for now as I'm not really testing the customer collection, but this could be improved in the future.